### PR TITLE
WIP :  Change channel name

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

**add description**

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- API documentation added: please review the Wiki page [Writing Documentation for the API](https://github.com/uyuni-project/uyuni/wiki/Writing-documentation-for-the-API) if you have any changes to API documentation.
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- No tests: already covered
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/support/constants.rb [hunk 1]
```diff
@@ -358,7 +358,7 @@
                             'SLE-Micro-5.3-Pool for x86_64' => 'sle-micro-5.3-pool-x86_64',
                             'SLE-Micro-5.4-Pool for x86_64' => 'sle-micro-5.4-pool-x86_64',
                             'almalinux9 for x86_64' => 'no-appstream-alma-9-result-almalinux9-x86_64',
-                            'Fake-Base-Channel' => 'fake-base-channel',
+                            'Fake-Base-Channel' => 'fake-base-channel-suse-like',
```
Comment (srbarrios) [inline testsuite/features/support/constants.rb:361]: Fake-Base-Channel-Suse-Like

**Review by nodeg — COMMENTED** [review]
I would write SUSE the correct way (`SUSE`), with all capitals where possible. In the other cases, I would write it all lowercase (`suse`).

**Comment by srbarrios** [comment]
> I would write SUSE the correct way (`SUSE`), with all capitals where possible. In the other cases, I would write it all lowercase (`suse`).

I would try to keep common namimg for all the channels, if we have channels in camel case and others with some words in capital letters might cause future mistakes.

**Comment by srbarrios** [comment]
Please double-check there are no other ruby files using the old channel label names.
For example, this might need to be update too:
```
Then(/^I should get the test base channel$/) do
  arch, _code = get_target('server').run('uname -m')
  arch.chomp!
  channel = if arch != 'x86_64'
              'fake-base-channel-i586'
            else
              'fake-base-channel'
            end
  log "result: #{@result}"
  assert(@result['channel_labels'].include?(channel))
end
```

**Comment by srbarrios** [comment]
We can close this draft, in favor of https://github.com/uyuni-project/uyuni/pull/7865 as discussed in https://suse.slack.com/archives/C02D12TNYLS/p1699949449154859?thread_ts=1699892428.469269&cid=C02D12TNYLS

Thanks for catching this Maxime.

**Pull request closed without merging**